testing/test_mark.py

```diff
@@ -1048,6 +1048,31 @@ def test():
     assert result.ret == ExitCode.INTERRUPTED
 
 
+def test_paramset_empty_no_idfunc(
+    pytester: Pytester, monkeypatch: pytest.MonkeyPatch
+) -> None:
+    p1 = pytester.makepyfile(
+        """
+        import pytest
+
+        def idfunc(value):
+           raise ValueError()
```

Comment:
IIUC `idfunc` is no longer being called when parametrize receives an empty list, is that it?

Reply:
correct , the exception here is to ensure its never getting triggered